docs/how_to_use/how_to_build.md

```diff
@@ -296,6 +306,21 @@ build:cuda --action_env=PATH=/usr/bin:/usr/local/bin
 build:cuda --host_action_env=PATH=/usr/bin:/usr/local/bin
 ```
 
+#### Update CUDA Compute Capabilities
```

Comment:
I thought about how kind the document should be to users. I think it would be okay to add a link like this, what do you think? I referred to the [TensorFlow documentation](https://www.tensorflow.org/install/pip#hardware_requirements).

```suggestion
#### Update CUDA [Compute Capabilities](https://developer.nvidia.com/cuda-gpus)
```